pkg/cliplugins/workspace/plugin/kubeconfig.go

```diff
@@ -109,7 +108,18 @@ func (kc *KubeConfig) ensureWorkspaceDirectoryContextExists(options *Options) (*
 			if err != nil {
 				return nil, err
 			}
-			workspaceDirectoryCluster.Server = fmt.Sprintf("%s://%s:%d%s", currentServerURL.Scheme, currentServerURL.Hostname(), workspacecmd.SecurePortDefault, defaultWorkspaceDirectoryApiServerPath)
+			orgClusterName := "root:default"
+			clusterIndex := strings.Index(currentServerURL.Path, "/clusters/")
+			if clusterIndex >= 0 {
+				orgClusterName = currentServerURL.Path[clusterIndex+10:]
+			}
+
+			scope := workspaceregistry.PersonalScope
+			if orgClusterName == helper.RootCluster {
+				scope = workspaceregistry.OrganizationScope
+			}
+
+			workspaceDirectoryCluster.Server = fmt.Sprintf("%s://%s:%d%s/%s/%s", currentServerURL.Scheme, currentServerURL.Hostname(), workspacecmd.SecurePortDefault, workspacebuilder.DefaultRootPathPrefix, orgClusterName, scope)
```

Comment:
time-💣 

We have to fix this quickly. It limits our ability to choose another deployment topology as we would break the cli plugin for everybody.

Reply:
related to this issue, which should be fixed in a next PR: https://github.com/kcp-dev/kcp/issues/638